NPE when running /view as a player

```
[15:41:02 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'view' in plugin BestViewDistance v11.9
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:48) ~[patched_1.13.2.jar:git-Paper-588]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:151) ~[patched_1.13.2.jar:git-Paper-588]

        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:734) ~[patched_1.13.2.jar:git-Paper-588]
        at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1782) ~[patched_1.13.2.jar:git-Paper-588]
        at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1590) ~[patched_1.13.2.jar:git-Paper-588]
        at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[patched_1.13.2.jar:git-Paper-588]
        at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:5) ~[patched_1.13.2.jar:git-Paper-588]
        at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:15) ~[patched_1.13.2.jar:git-Paper-588]
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[?:1.8.0_201]
        at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:1.8.0_201]
        at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:108) ~[patched_1.13.2.jar:git-Paper-588]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1011) ~[patched_1.13.2.jar:git-Paper-588]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-588]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:938) ~[patched_1.13.2.jar:git-Paper-588]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:836) ~[patched_1.13.2.jar:git-Paper-588]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_201]
Caused by: java.lang.NullPointerException
        at me.lxct.bestviewdistance.functions.util.Misc.replacePlaceHolders(Misc.java:22) ~[?:?]
        at me.lxct.bestviewdistance.commands.Commands.commandHelp(Commands.java:26) ~[?:?]
        at me.lxct.bestviewdistance.commands.ViewCommand.onCommand(ViewCommand.java:19) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[patched_1.13.2.jar:git-Paper-588]
        ... 15 more
```

Hey gray, what a coincident we happen to be testing this plugin at the same time. :P

Compiled as of commit https://github.com/Kugge/BestViewDistance/commit/8e335e8d61ba77dd4e1c83dfda28e23ac98771f9
Paper dev 588
PlaceholderAPI dev 58
@Lxct 
```
[14:13:54] [Server thread/INFO]: mibby issued server command: /view
[14:13:54] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'view' in plugin BestViewDistance v11.9
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:48) ~[patched_1.13.2.jar:git-Paper-588]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:151) ~[patched_1.13.2.jar:git-Paper-588]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:734) ~[patched_1.13.2.jar:git-Paper-588]
	at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1782) ~[patched_1.13.2.jar:git-Paper-588]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1590) ~[patched_1.13.2.jar:git-Paper-588]
	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[patched_1.13.2.jar:git-Paper-588]
	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:5) ~[patched_1.13.2.jar:git-Paper-588]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:15) ~[patched_1.13.2.jar:git-Paper-588]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:108) ~[patched_1.13.2.jar:git-Paper-588]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1011) ~[patched_1.13.2.jar:git-Paper-588]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-588]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:938) ~[patched_1.13.2.jar:git-Paper-588]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:836) ~[patched_1.13.2.jar:git-Paper-588]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NullPointerException
	at me.lxct.bestviewdistance.functions.util.Misc.replacePlaceHolders(Misc.java:22) ~[?:?]
	at me.lxct.bestviewdistance.commands.Commands.commandHelp(Commands.java:26) ~[?:?]
	at me.lxct.bestviewdistance.commands.ViewCommand.onCommand(ViewCommand.java:19) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[patched_1.13.2.jar:git-Paper-588]
	... 15 more
```

--- a/src/main/java/me/lxct/bestviewdistance/commands/ViewCommand.java
+++ b/src/main/java/me/lxct/bestviewdistance/commands/ViewCommand.java
@@ -2,6 +2,8 @@ package me.lxct.bestviewdistance.commands;
 
 import me.lxct.bestviewdistance.functions.util.Misc;
 import me.lxct.bestviewdistance.functions.data.Variable;
+
+import org.bukkit.ChatColor;
 import org.bukkit.command.Command;
 import org.bukkit.command.CommandExecutor;
 import org.bukkit.command.CommandSender;
@@ -14,6 +16,13 @@ public class ViewCommand implements CommandExecutor {
 
     @Override
     public boolean onCommand(final CommandSender sender, final Command cmd, final String label, final String[] args) {
+        if (!(sender instanceof Player)) {
+            sender.sendMessage(ChatColor.RED + "Sadly, this requires a player.");
+            return true;
+        }
+
+        playerData = onlinePlayers.get(sender);
+
         if (cmd.getName().equalsIgnoreCase("view")) {
             if (args.length == 0) {
                 commandHelp(sender);
@@ -24,26 +33,23 @@ public class ViewCommand implements CommandExecutor {
                     commandPing(args, sender);
                     commandView(args, sender);
                 }
+
                 if (sender.hasPermission("view.reload")) {
                     commandReload(args, sender);
                 }
             }
         } else if (cmd.getName().equalsIgnoreCase("vdist") && sender.hasPermission("view.info")) {
-            if (sender instanceof Player) {
-                playerData = onlinePlayers.get(sender);
-                sender.sendMessage(colorize(Misc.replacePlaceHolders(vdistLine1)));
-                sender.sendMessage(colorize(Misc.replacePlaceHolders(vdistLine2)));
-                sender.sendMessage(colorize(Misc.replacePlaceHolders(vdistLine3)));
-                if (!Variable.hideVdistLine4) {
-                    sender.sendMessage(colorize(Misc.replacePlaceHolders(vdistLine4)));
-                }
+            sender.sendMessage(colorize(Misc.replacePlaceHolders(vdistLine1)));
+            sender.sendMessage(colorize(Misc.replacePlaceHolders(vdistLine2)));
+            sender.sendMessage(colorize(Misc.replacePlaceHolders(vdistLine3)));
+
+            if (!Variable.hideVdistLine4) {
+                sender.sendMessage(colorize(Misc.replacePlaceHolders(vdistLine4)));
             }
         } else if (cmd.getName().equalsIgnoreCase("vping") && sender.hasPermission("view.info")) {
-            if (sender instanceof Player) {
-                playerData = onlinePlayers.get(sender);
-                sender.sendMessage(colorize(Misc.replacePlaceHolders(vping)));
-            }
+            sender.sendMessage(colorize(Misc.replacePlaceHolders(vping)));
         }
+
         return true;
     }
 }

